cmd/podman/build.go

```diff
@@ -15,16 +10,26 @@ import (
 	"github.com/projectatomic/buildah/pkg/parse"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
 )
 
 var (
+	layerFlags = []cli.Flag{
+		cli.BoolTFlag{
+			Name:  "layers",
+			Usage: "cache intermediate layers during build. Use BUILDAH_LAYERS environment variable to override. ",
+		},
+	}
```

Comment:
Not an issue for this PR, but a @rhatdan question.  Since both Buildah and Podman are using this variable, it seems odd to be named for one over the other.  Could we change it to a generic variable, perhaps CONTAINER_LAYERS?

Reply:
We just BUILDAH_ in a couple of other places and we have already shipped/supported the flags in BUILDAH.  I don't think we need to hide the fact that their is a relationship between podman build and BUILDAH.

Reply:
FWIW, wasn't trying to hide the relationship, just trying to make the environment variable more generic.